client/layout/sidebar/menu.jsx

```diff
@@ -0,0 +1,10 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+const SidebarMenu = ( { children } ) => (
+	<li className="sidebar__menu">{ children }</li>
```

Comment:
Makes sense. It would also allow us to eventually drop that markup if we want without having to touch the files.